system variables: add `tidb_log_file_max_days` (#6773)
pingcap/docs: system-variables.md

- Default value: `tikv,tiflash,tidb`
- This variable is used to set the storage engine list that TiDB can use when reading data.

### tidb_log_file_max_days <span class="version-mark">New in v5.3</span>

- Scope: SESSION
- Default value: `0`

> - This variable is used to adjust max days of logger on the current TiDB instance. It defaults to the value specified in the config file, refer [`max-days`](/tidb-configuration-file.md#max-days). Changing the variable will only replace the logger of the current TiDB instance. The variable will be reset after an restart and the config file will not be affected.

Reviewer: ```suggestion
- This variable is used to adjust the maximum days of logger on the current TiDB instance. Its value defaults to the value of the [`max-days`](/tidb-configuration-file.md#max-days) configuration in the configuration file. Changing the variable value only affects the current TiDB instance. After TiDB is restarted, the variable value is reset and the configuration value is not affected.
```
